### DISCUSSION TITLE
How to make eww go below fullscreen apps

### DISCUSSION THREAD

**Discussion by musjj** [elkowar/eww#710] [Q&A]
Right now my eww window stays over fullscreen apps like mpv and YouTube (in the browser). How do I make it go below it?
I tried adding `:stacking "bg"` to my window definition but it's not working. I'm using bspwm.

EDIT: Looks like this is an ongoing issue on certain WMs: https://github.com/elkowar/eww/issues/178

**Comment by quiltedstars** [comment]
Replace :focusable "false" with :wm-ignore "false"

**Reply by musjj** [reply]
Still doesn't work for me. Are you using bspwm? If so, can I see your dotfiles?

**Reply by quiltedstars** [reply]
@musjj I'm using xfce, which was listed as having a similar issue. I'm using adi1090x's widgets, but here's the code I changed per each widget to get them behind my apps:
```
;; system
(defwindow system :stacking "bg" :wm-ignore "false" :screen 1 
	    :geometry (geometry :x 150 :y 605 :width 350 :height 325)
					(system))
```

**Reply by Tsukimid** [reply]
have you solved this issue yet?

**Reply by musjj** [reply]
No, it's still an ongoing issue tracked in the link I posted above. There's apparently a working hack (with caveats) posted by @jrachele in the thread, but I haven't tried it yet.

**Reply by Tsukimid** [reply]
> No, it's still an ongoing issue tracked in the link I posted above. There's apparently a working hack (with caveats) posted by @jrachele in the thread, but I haven't tried it yet.

yeah, it gets treated as a window.

**Comment by webbb82** [comment]
Ijust realized this a few hours ago. I couldn't put my finger on why eww seemed in the way. ya oher bars go under or  they stay above al other window like the bottom edge is the new top of the window
